FordLabs/PeopleMover
# GitHub templates

## Issue
Connects: N/A

## What was done
- [x] Added this pull request template
- [x] Fixed lint errors that were auto fixable

## How to test
1. Ensure that the pull request template is a good format for this team and that it is in the correct github template format.
